File: apollo/src/main/java/io/muun/apollo/domain/errors/LnInvoiceNotSupportedError.java
package io.muun.apollo.domain.errors;


import io.muun.apollo.external.UserFacingErrorMessages;

public class LnInvoiceNotSupportedError extends UserFacingError {

    public LnInvoiceNotSupportedError() {
        super(UserFacingErrorMessages.INSTANCE.lnInvoiceNotSupported());
    }

    public LnInvoiceNotSupportedError(Throwable cause) {
        super(UserFacingErrorMessages.INSTANCE.lnInvoiceNotSupported(), cause);
    }
}

File: apollo/src/main/java/io/muun/apollo/domain/errors/LnInvoiceNotSupportedError.kt
package io.muun.apollo.domain.errors


import io.muun.apollo.external.UserFacingErrorMessages

class LnInvoiceNotSupportedError: UserFacingError {

    constructor():
        super(UserFacingErrorMessages.INSTANCE.lnInvoiceNotSupported())

    constructor(cause: Throwable):
        super(UserFacingErrorMessages.INSTANCE.lnInvoiceNotSupported(), cause)
}
